db/migrate/20230316034707_disable_inherits_tax_category_from_per_order_enterprise_fees.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class DisableInheritsTaxCategoryFromPerOrderEnterpriseFees < ActiveRecord::Migration[6.1]
+  class EnterpriseFee < ApplicationRecord
+    has_many :calculator, class_name: "Spree::Calculator", foreign_key: :calculable_id
+  end
+
+  module Spree
+    class Calculator < ApplicationRecord
+      self.table_name = 'spree_calculators'
+    end
+  end
+
+  def change
```

Comment:
Migrations have two actions: `rake db:migrate` (perform the migration) and `rake db:rollback (undo the migration if possible). 

This method migrates the data to the new state, so it should only run on `migrate`. 
In this case we're not losing any information, so we don't need a rollback (I think the `inherits_tax_category` did nothing before).

The `change` method runs for both `migrate` and `rollback`. To only run on `migrate`, we can just define `up`:

```diff
-  def change
+  def up
```

This is only for future reference. It doesn't really matter in this case if it runs on rollbacks so I don't suggest we bother updating the code.